devsetup/tripleo/tripleo_install.sh

```diff
@@ -77,7 +88,7 @@ openstack overcloud deploy --stack overcloud \
     -e /home/zuul/hostnamemap.yaml -e /usr/share/openstack-tripleo-heat-templates/environments/docker-ha.yaml \
     -e /home/zuul/containers-prepare-parameters.yaml -e /usr/share/openstack-tripleo-heat-templates/environments/podman.yaml \
     -e /usr/share/openstack-tripleo-heat-templates/environments/low-memory-usage.yaml \
-    -e /usr/share/openstack-tripleo-heat-templates/environments/debug.yaml --validation-warnings-fatal ${CEPH_OVERCLOUD_ARGS} \
```

Comment:
green run for both ceph/non ceph job at https://review.rdoproject.org/r/c/testproject/+/52732/14#message-f95bbae393599b1c0505fe5d629cabc84623b326